src/main/java/io/zohoinventory/ItemAPI.java

```diff
@@ -0,0 +1,41 @@
+package io.zohoinventory;
+
+import io.zohoinventory.models.authenticationapi.AccessCredentials;
+import io.zohoinventory.models.itemapi.ItemWrapper;
+import io.zohoinventory.models.itemapi.ItemsWrapper;
+
+import java.net.URI;
+import java.net.http.HttpRequest;
+import java.util.HashMap;
+
+import static io.zohoinventory.constantcode.ConstantCodes.*;
+
+public class ItemAPI extends ZohoInventorySdk{
+
+    public ItemAPI(AccessCredentials accessCredentials){
+        super(accessCredentials);
+    }
+
+    public ItemsWrapper getItems(HashMap<String, String> params) {
+        refreshAccessToken();
+        URI uri = URI.create(API_BASED_END_POINT
```

Comment:
declare the variable String uri = API_BASED_END_POINT + VERSION + ITEM_ENDPOINT;
and pass it into URI.create